.github/workflows/ci.yml

```diff
@@ -58,16 +58,3 @@ jobs:
         run: |
           bun install
           bun test --coverage
-
-  lint:
-    runs-on: ubuntu-latest
-    steps:
```

Comment:
Forgot to add back linting that uses biome?

Reply:
@marcomariscal

Reply:
`bun run build` handles the linting, so it would be redundant i think.

can add back in if you feel like we should have both explicitly.

Reply:
yes, let's separate them. Bun run build should only run `bun tsc`

Reply:
https://github.com/ScopeLift/stealth-address-sdk/pull/56/commits/4c47f1ab8dcfa8f6e676cd5442756362c1250a78